aspnetcore/signalr/version-differences.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Differences between SignalR and ASP.NET Core SignalR
+author: rachelappel
+description: Differences between SignalR and ASP.NET Core SignalR 
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+uid: signalr/version-differences
+---
+
+# Differences between SignalR and ASP.NET Core SignalR
+
+ASP.NET Core SignalR is not compatible with clients or servers for ASP.NET SignalR. This article details the features which have been removed or changed in the ASP.NET Core SignalR.
+
+## Feature differences
+
+### Automatic reconnects
+
+Automatic reconnects are no longer supported. Previously, SignalR tried to reconnect to the server if the connection was dropped. Now, if the client is disconnected the user must explicitly start a new connection if they want to reconnect.
```

Comment:
~~Now, if the client is disconnected the user must explicitly start a new connection if they want to reconnect.~~
Now, if the client is disconnected, the user must explicitly start a new connection to reconnect.